hydra-kafka10/src/main/java/com/addthis/hydra/kafka/bundle/ConsumerUtils.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.addthis.hydra.kafka.bundle;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.PartitionInfo;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class ConsumerUtils {
+
+    private static final Logger log = LoggerFactory.getLogger(ConsumerUtils.class);
+
+    public static Map<String, List<PartitionInfo>> getTopicsMetadata(KafkaSource kafkaSource, List<String> topics) {
```

Comment:
this function should take just the consumerProperties as an argument, instead of a KafkaSource